firmware_mod/scripts/detectionTracking.sh

```diff
@@ -84,40 +84,23 @@ fi
 # Basic algorithm to calculate the movement
 # Not optimized, if you have ideas to simplify it ...
```

Comment:
Perhaps this comment can now be changed ;-)

Reply:
It could maybe be optimized better